app/helpers/admin/views_helper.rb

```diff
@@ -0,0 +1,5 @@
+module Admin::ViewsHelper
+  def check_url view
+    view.id.present? ? admin_view_path(view) : admin_views_path(view)
```

Comment:
```suggestion
    view.id.present? ? admin_view_path(view) : admin_views_path
```
y chi la ntn co ma
k doc ki cau hoi ah

Reply:
Hehe vậng ạ. Nó không cần tham số truyền vào ạ

Reply:
Em sửa lại rồi ạ